modules/power-monitoring-fips-support.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * power_monitoring/power-monitoring-overview.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="power-monitoring-fips-support_{context}"]
+= About FIPS compliance for {PM-operator}
+
+Starting with version 0.4, {PM-operator} for Red Hat OpenShift is FIPS compliant. When deployed on an {product-title} cluster in FIPS mode, it uses {op-system-base-full} cryptographic libraries validated by NIST.
```

Comment:
Consider expanding NIST here for its first instance.

I think it's -- National Institute of Standards and Technology (NIST)

Reply:
I am keeping the attribute as it is as I don't think we need to mention the "for Red Hat OpenShift" part always.